downstream/modules/aap-hardening/ref-architecture.adoc

```diff
@@ -7,11 +7,11 @@
+For large-scale production environments with availability requirements, this guide recommends deploying the components described in section 2.1 of this guide using the instructions in the xref:ref-architecture_{context}[reference architecture] documentation for {PlatformName} on {RHEL}. While some variation may make sense for your specific technical requirements, following the reference architecture results in a supported production-ready environment.
 
 .Reference architecture overview
-image::aap-ref-architecture-322.png[Reference architecture for an example setup of an Ansible Automation Platform deployment for large scale production environments]
+image::aap-ref-architecture-322.png[Reference architecture for an example setup of an {PlatformNameShort} deployment for large scale production environments]
 
-{EDAName} is a new feature of {PlatformNameShort} {PlatformVers} that was not available when the reference architecture detailed in Figure 1: Reference architecture overview was originally written. Currently, the supported configuration for {EDAName} is a single {EDAcontroller} with a dedicated internal database. For an organization interested in {EDAName}, the recommendation is to install according to the configuration documented in the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/assembly-platform-install-scenario#ref-single-eda-controller-with-internal-db_platform-install-scenario[Ansible Automtion Platform Installation Guide]. This document will provide additional clarifications when {EDAName} specific hardening configuration is required.
+{EDAName} is a new feature of {PlatformNameShort} {PlatformVers} that was not available when the reference architecture detailed in Figure 1: Reference architecture overview was originally written. Currently, the supported configuration for {EDAName} is a single {EDAcontroller} with a dedicated internal database. For an organization interested in {EDAName}, the recommendation is to install according to the configuration documented in the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_installation_guide/assembly-platform-install-scenario#ref-single-eda-controller-with-internal-db_platform-install-scenario[{PlatformNameShort} Installation Guide]. This document will provide additional clarifications when {EDAName} specific hardening configuration is required.
```

Comment:
That link shouldn't exist anymore, the example has gone.

The document will provide -> The document provides